### PR TITLE
Updated setup.py version requirements to support 3.6.8 for Windows

### PR DESCRIPTION
**Pull request by madhavajay**
- 3.6.8 was the final binary release of 3.6.x for Windows
- Added PR Tests to check CI installs opacus on different combinations

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Docs change / refactoring / dependency upgrade

## Motivation and Context / Related issue
To support 3.6.x on all platforms it makes sense to support 3.6.8 which was the last binary release for Windows of the 3.6.x range.

## How Has This Been Tested (if it applies)
This PR includes GitHub Action CI to test that it works.

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the **CONTRIBUTING** document and completed the CLA (see **CONTRIBUTING**).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -81,5 +82,5 @@
         "Programming Language :: Python :: 3 :: Only",
         "Topic :: Scientific/Engineering",
     ],
-    python_requires=">=3.6.9",
+    python_requires=f">={REQUIRED_MAJOR}.{REQUIRED_MINOR}.{REQUIRED_MICRO}",
```
Comment (Darktex) [inline setup.py:85]: Ah, nice catch! 👍

File: .github/workflows/pr-tests.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# This workflow makes sure edge case versions work
```
Comment (Darktex) [inline .github/workflows/pr-tests.yml:1]: It's safe to remove this file, we have circleci to run these tests :)

**Review by Darktex — CHANGES_REQUESTED** [review]

**Review by Anonymani — APPROVED** [review]

**Comment by madhavajay** [comment]
@Darktex The GitHub actions file got renamed, just to clarify do you want me to remove the `.github/workflows/test-installation.yml` file and the associated scripts that it uses?
